packages/openapi-react-query/src/index.ts

```diff
@@ -217,15 +217,11 @@ export default function createClient<Paths extends {}, Media extends MediaType =
       useSuspenseQuery(queryOptions(method, path, init as InitWithUnknowns<typeof init>, options), queryClient),
     useInfiniteQuery: (method, path, init, options, queryClient) => {
       const { pageParamName = "cursor", ...restOptions } = options;
-
+      const { queryKey } = queryOptions(method, path, init);
```

Comment:
This _teeeechnically_ adds a modicum of work, but barely. The other methods were already using this; we’re just dogfooding it.

We probably could improve the `queryFn` helper to work for `useInfiniteQuery` but I didn’t want to make too big a change.

Reply:
Want to make a issue for it?